docs/src/modules/components/overview/charts/advancedCharts/HeatmapDemo.tsx

```diff
@@ -1,21 +1,21 @@
 /* eslint-disable material-ui/no-hardcoded-labels */
 import * as React from 'react';
-import { interpolateBlues } from 'd3-scale-chromatic';
+import { interpolateReds } from 'd3-scale-chromatic';
```

Comment:
Are you sure about this color? to display heat wave, or other temperature related stuff why not, but the red to cound the number of bycicle feels weird

![image](https://github.com/user-attachments/assets/01b110ce-04b3-4068-8af6-ebe6150cc45b)